packages/amplify_authenticator/lib/src/strings/navigation_strings.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:amplify_authenticator/src/text_customization/auth_strings_resolver.dart';
+import 'package:amplify_authenticator/src/text_customization/button_resolver.dart';
+
+/// The class for displaying navigation related labels
+class NavigationStrings {
+  /// The optional class that customers can pass to create custom values
+  NavigationResolver? resolver;
+
+  static const _navigate_signin = "Sign In";
```

Comment:
As I said before, I think these values should come from a default English resolver.

Reply:
As we discussed, we can hold off on this.